BUILD_macos.md

```diff
@@ -78,8 +78,8 @@ PRs with corrections and updates are welcome!
   ```
 
 * To install tools for building the UI:
-  * `brew install node yarn`
-  * Currently, [`yarn`](https://classic.yarnpkg.com/en/docs/install) (< 2.0.0) is required
+  * `brew install node`
+  * `corepack enable`
```

Comment:
Though pnpm docs say this:

> If you installed Node.js using Homebrew, you'll need to install corepack separately:
>
> `brew install corepack`
>
> This will automatically install pnpm on your system.

Should it just say `brew install node corepack` then?

Reply:
Uh that's correct, I just tried a fresh installation :/
I suppose it didn't work that way in the past, because I didn't install corepack with brew.

Changed to `brew install node corepack`.